File: src/main/java/org/tdf/lotusvm/types/LongBuffer.java
package org.tdf.lotusvm.types;

import java.io.Closeable;

public interface LongBuffer extends Closeable {
    long get(int index);

    void set(int index, long val);

    int size();

    void push(long val);

    void setSize(int size);

    default void close() {
    }
}

File: src/main/java/org/tdf/lotusvm/types/LongBuffer.kt
package org.tdf.lotusvm.types

interface LongBuffer : AutoCloseable {
    operator fun get(index: Int): Long
    operator fun set(index: Int, `val`: Long)
    fun size(): Int
    fun push(value: Long)
    fun setSize(size: Int)
    override fun close() {}
}